src/documents/tests/test_api.py

```diff
@@ -1362,6 +1363,27 @@ def test_tag_text_color(self):
             "#000000",
         )
 
+    def test_ui_settings(self):
+        test_user = User.objects.create_superuser(username="test")
+        self.client.force_login(user=test_user)
+
+        response = self.client.get(f"/api/ui_settings/", format="json")
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.data["settings"],
+            [],
+        )
+
+        UiSettings.objects.create(
+            user=test_user,
+            settings='{"dark_mode":[{"enabled":"true"}]}',
+        )
+        response = self.client.get(f"/api/ui_settings/", format="json")
+        self.assertNotEqual(
+            response.data["settings"],
+            [],
```

Comment:
To be honest I would love to write more detailed tests but I had a hell of a time with pytest. 

Without any modifications on my machine pytest fails every api test saying it's not authorized, even though they work on ci and presumably for others. Since I had so much trouble I kept the tests quite simple. I would love to figure out how to get it working properly and then I could make these better, or of course if others would like to add tests themselves that would be great too.

Reply:
That's strange, I don't recall needing to do anything special to get those tests working.

Reply:
Yea, it drove me a bit mad. I find pytest kind of opaque tbh, like it says it should "just work" but how bizarre it works on ci and not locally.

If I can muster the strength I will try to dive in again 😀 ...

Reply:
Since I'm able to run them (and have no idea what might be wrong with the setup), I made some adjustments to the testing.  Hopefully nothin broke in the frontend?

Reply:
Sweet thank you!